docs/api/ShallowWrapper/matchesElement.md

```diff
@@ -21,16 +21,16 @@ render tree.
 
 
 ```jsx
-const MyComponent = React.createClass({
+class MyComponent extends React.Component {
   handleClick() {
     ...
```

Comment:
this component doesn't really work now that it's a class because methods aren't auto bound, so if `this` is used in handleClick it won't work. 

do we want to have an "invalid" component as an example in the docs?